src/AbstractPPL.jl

```diff
@@ -7,6 +7,8 @@ export VarName, getsym, getlens, inspace, subsumes, varname, vsym, @varname, @vs
 # Abstract model functions
 export AbstractProbabilisticProgram, condition, decondition, logdensityof, densityof
 
+# GraphInfo
+export GraphInfo, Model, dag, nodes
```

Comment:
The name `Model` and, to a smaller degree, `nodes` and `dag` seem to general to be exported.

Reply:
No objections to getting rid of `nodes` and `dag`. I've put `GraphInfo` into a submodule `GraphPPL` that isn't exported, as per @phipsgabler suggestion. Does this mitigate concerns about directly exporting Model?